src/sync/deployment.test.ts

```diff
@@ -206,10 +206,24 @@ describe("sync/deployments", () => {
 			commitsFromDate: "2023-01-01T00:00:00Z"
 		});
 
+		interface DeploymentEntity {
```

Comment:
I think we already have these types here: https://github.com/atlassian/github-for-jira/blob/NONE-add-mem-stats-and-improve-logging/src/github/client/github-queries.ts#L533-L533